CRM/Queue/Queue/Memory.php

```diff
@@ -110,12 +110,35 @@ public function createItem($data, $options = []) {
   }
 
   /**
-   * Determine number of items remaining in the queue.
-   *
-   * @return int
+   * @param string $name
+   * @return int|float|null
+   * @see \CRM_Queue_Queue::getStatistic()
    */
-  public function numberOfItems() {
-    return count($this->items);
+  public function getStatistic(string $name) {
+    $ready = function(): int {
```

Comment:
Hmm - this inline function threw me at first - would be more readable as a separate function but OK - it's a short enough function it doesn't matter

Reply:
PHP visibility levels:

* `public`
* `protected`
* `private`
* `so_private_you_can_t_even_see_it`

Reply:
lol - yes it is VERY private